README.md

```diff
@@ -8,6 +8,9 @@
 [![](https://dcbadge.vercel.app/api/server/primitive?style=flat)](https://discord.gg/primitive)
 [![Twitter Badge](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/primitivefi)
 
+
```

Comment:
I am sure we can improve on this in the future, but seemed like a step in the right direction. We should probably review more thoroughly before merging.